sec/review.tex

```diff
@@ -210,11 +215,11 @@ \subsubsection{Centroid-based clustering}\label{subsubsec:kmeans}
 
 At this point, the survey returns to the effect of outliers on Lloyd-like
 algorithms, especially \(k\)-means, and the literature around reducing that
-effect.~\cite{Olukanmi2017} introduces the \(k\)-means\# algorithm --- an
-extension to \(k\)-means that embeds an outlier detection feature which relies
-solely on the underlying structures from the original \(k\)-means algorithm.
-The benefit of handling adverse noise in this way is twofold: first, it
-preserves the computational efficiencies that come with using the mean (as
+effect.~\cite{Olukanmi2017} introduces the \(k\)-means\# (sharp) algorithm ---
```

Comment:
@drvinceknight this wasn't a typo, but does this make it clearer?

Reply:
No I don't think it needs the `(sharp)` as long as that's not a typo it's not needed. If you wanted to you could go with:

```
\(k\)-means sharp, denoted \(k\)-means\#
```

?

Reply:
In the title of the article, they refer to it as k-means-sharp, and later as k-means#, so I'll just use the first.

Reply:
Perfect.

Reply:
Fixed in 7bae33b